bootstrap/templates/addons/kube-prometheus-stack/app/helmvalues.yaml.j2

```diff
@@ -94,8 +94,11 @@ data:
         enabled: true
         ingressClassName: internal
         annotations:
-          hajimari.io/appName: Prometheus
-          hajimari.io/icon: simple-icons:prometheus
+          gethomepage.dev/enabled: "true"
+          gethomepage.dev/icon: prometheus.png
+          gethomepage.dev/name: Prometheus
+          gethomepage.dev/group: Observability
+          gethomepage.dev/widget.type: prometheus
```

Comment:
I wonder if we should start wrapping the annotation blocks with 

```
{% if homepage.enabled | default(false) %}
...
{% endif %}
```

Reply:
Tend to agree here, just to help keep things a little cleaner, especially if someone decides they don't want to use homepage.

Reply:
100% we should do that. I'll add it. Great suggestion.